### PR TITLE
Feature/Breaking Change: add support for right-aligned icons

### PR DESCRIPTION
**Pull request by vkkostin**
## Problem
We need to implement an alert-style icon with a tooltip for `Current Device Location` menu items if the Google API key is missing or invalid.
Screen Shot 2022-02-01 at 12 31 51 PM

## Solution
Use the `getIcon` utility function that is currently used for our standard left-aligned icons to render right-aligned icons. I folded in the only other interface that has a right-aligned icon, which is the locked font style options. The logic that renders font options will pass in a pre-styled component to render the lock icon, rendering the `.multi-menu-icon-lock` redundant.
Screen Shot 2022-02-01 at 2 46 04 PM

### CODE REVIEW

**Comment by vkkostin** [comment]
[Get Device Location as address in Magic Text and Property Dropdowns](https://trello.com/c/zjeYIM35/43-get-device-location-as-address-in-magic-text-and-property-dropdowns)

**Comment by tobyjsullivan** [comment]
Which part of the change breaks backward compatibility? Is it that the `lock` needs to be specified explicitly rather than showing automatically when `lock` prop is present? Any other reasen?

File: styles.css [hunk 1]
```diff
@@ -432,21 +442,4 @@
...
...oZWQuZXhwYW5kLWRvd25cbiAgPiAubXVsdGktbWVudS1pbm5lci13cmFwcGVyXG4gID4gLm11bHRpLW1lbnUge1xuICBib3JkZXItdG9wLWxlZnQtcmFkaXVzOiAwO1xuICBib3JkZXItdG9wLXJpZ2h0LXJhZGl1czogMDtcbiAgbWFyZ2luLXRvcDogLTFweDtcbn1cblxuLm11bHRpLW1lbnUtd3JhcHBlci1hdHRhY2hlZC5leHBhbmQtdXBcbiAgPiAubXVsdGktbWVudS1pbm5lci13cmFwcGVyXG4gID4gLm11bHRpLW1lbnUge1xuICBib3JkZXItYm90dG9tLWxlZnQtcmFkaXVzOiAwO1xuICBib3JkZXItYm90dG9tLXJpZ2h0LXJhZGl1czogMDtcbiAgbWFyZ2luLWJvdHRvbTogLTFweDtcbn1cblxuLm11bHRpLW1lbnUtd3JhcHBlci1hdHRhY2hlZCA+IC5tdWx0aS1tZW51LWlubmVyLXdyYXBwZXI6YmVmb3JlIHtcbiAgY29udGVudDogJyc7XG4gIGRpc3BsYXk6IGJsb2NrO1xuICBwb3NpdGlvbjogYWJzb2x1dGU7XG4gIHRvcDogLTRweDtcbiAgbGVmdDogMDtcbiAgcmlnaHQ6IDA7XG4gIGhlaWdodDogNHB4O1xuICBiYWNrZ3JvdW5kOiAjZjVmNWY1O1xuICB6LWluZGV4OiAxO1xufVxuXG4ubXVsdGktbWVudS13cmFwcGVyLWF0dGFjaGVkLmV4cGFuZC11cCA+IC5tdWx0aS1tZW51LWlubmVyLXdyYXBwZXI6YmVmb3JlIHtcbiAgdG9wOiBhdXRvO1xuICBib3R0b206IC02cHg7XG4gIGhlaWdodDogN3B4O1xufVxuXG4ubXVsdGktc2VsZWN0LW1lbnUubXVsdGktbWVudS10cmlnZ2VyLmV4cGFuZGVkOmFmdGVyIHtcbiAgZGlzcGxheTogbm9uZTtcbn1cblxuLm11bHRpLXNlbGVjdC1tZW51LXBsYWNlaG9sZGVyLFxuLm11bHRpLW1lbnUtZW1wdHkge1xuICBjb2xvcjogIzllOWU5ZTtcbn1cblxuLm11bHRpLW1lbnUtZW1wdHkge1xuICBwYWRkaW5nOiAwIDEycHg7XG59XG4iXX0= */
```
Comment (nucks) [inline styles.css:445]: Looks good. What is this for? I've never seen this.

**Review by nucks — APPROVED** [review]

**Comment by vkkostin** [comment]
> Which part of the change breaks backward compatibility? Is it that the `lock` needs to be specified explicitly rather than showing automatically when `lock` prop is present? Any other reason?

Yes. In our case specifically, it would break the branding font dropdowns in the editor.

**Comment by tobyjsullivan** [comment]
If it's just the lock thing, would it be easier to add a fallback and preserve the backward compatibility?

For a fallback, I'm imagining something like:

```js
if (lock && !data.leftIcon) {
    data.leftIcon = 'lock'
}
```

Either way works. The breaking change seems fine if we're 100% sure it's just the font menu that needs to be updated.

**Pull request merged**